assets/js/webinar.js

```diff
@@ -45,4 +71,35 @@ $(function() {
             });
         });
     }
+
+    // Check that the analytics track function is available.
+    var analyticsAvailable = window.analytics && window.analytics.track &&
+                             (typeof window.analytics.track === "function");
+
+    if (webinarDetails && analyticsAvailable) {
+        // Parse the cookie and grab the UTM cookie values.
+        var cookies = parseCookie();
+        var utmCookie = parseUTMCookie(cookies["__utmzz"]);
+
+        // Listen for the HubSpot form to be loaded.
+        window.addEventListener('message', event => {
+            if(event.data.type === 'hsFormCallback' && event.data.eventName === 'onFormReady') {
+                var form = $(".hbspt-form form");
+                console.log(form);
```

Comment:
Was it intentional to keep the `console.log`?

Reply:
Nope, just a mistake after a long day.. Thanks for pointing this out.